resolvers/node/builtin_modules.rs

```diff
@@ -0,0 +1,103 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+pub trait IsBuiltInNodeModuleChecker: std::fmt::Debug {
+  /// e.g. `is_builtin_node_module("assert")`
+  fn is_builtin_node_module(&self, module_name: &str) -> bool;
+}
+
+/// An implementation of IsBuiltInNodeModuleChecker that uses
+/// the list of built-in node_modules that are supported by Deno
+/// in the `deno_node` crate (ext/node).
+#[derive(Debug)]
+pub struct DenoIsBuiltInNodeModuleChecker;
+
+impl IsBuiltInNodeModuleChecker for DenoIsBuiltInNodeModuleChecker {
+  #[inline(always)]
+  fn is_builtin_node_module(&self, module_name: &str) -> bool {
+    DENO_SUPPORTED_BUILTIN_NODE_MODULES
+      .binary_search(&module_name)
+      .is_ok()
+  }
+}
+
+/// Collection of built-in node_modules supported by Deno.
+pub static DENO_SUPPORTED_BUILTIN_NODE_MODULES: &[&str] = &[
+  // NOTE(bartlomieju): keep this list in sync with `ext/node/polyfills/01_require.js`
```

Comment:
Make sure this is reachable when a crate is imported - Deploy relies on this list

Reply:
Double checked and it is.